frontend/src/metabase/query_builder/components/NativeQueryEditor/DataSourceSelectors/DataSourceSelectors.jsx

```diff
@@ -177,11 +184,18 @@ const TableSelector = ({ database, readOnly, selectedTable, setTableId }) => (
 
 TableSelector.propTypes = TableSelectorPropTypes;
 
-const Placeholder = ({ query }) => (
-  <div className="ml2 p2 text-medium">
-    {t`This question is written in ${getNativeQueryLanguage(query.engine())}.`}
-  </div>
-);
+const Placeholder = ({ query, editorContext }) => {
+  if (editorContext === "action") {
```

Comment:
nit: Maybe `This action is written in `${language}.`?

Reply:
If we don't want to add new translations, maybe return just `language` here? And we can add a proper placeholder in master only